app/views/categories/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav class="">
+  <i class="fa-solid fa-chevron-left"><%= link_to 'Back', categories_path, class: 'your-link-class' %></i>
+</nav>
+<h1>New category</h1>
+
+<%= render "form", category: @category %>
+
+<br>
```

Comment:
- [ ] After clicking on the Submit button on the "Add a new category" page, It should redirect the users to the home page(categories page) on success as requested in the project requirements but it's redirecting users to the transaction page(expenses page) of the created category. Please make sure to fix this issue.

![image](https://github.com/Othniel2471/budget_app/assets/83803414/46a07d54-1aa7-4532-8e4c-da770c79775a)